<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StockBox</class>
 <widget class="QWidget" name="StockBox">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>90</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>50</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Stock Box</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#StockBox{
background-color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QTableView" name="stockBoxDataTableView">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>22</y>
     <width>764</width>
     <height>66</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>10</number>
   </attribute>
  </widget>
  <widget class="QPushButton" name="startInverseMatchingButton">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>0</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Matching</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startInverseSellingButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Sell</string>
   </property>
  </widget>
  <widget class="QPushButton" name="copyAskButton">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>0</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="copyBidButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>0</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startBuyingButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>0</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Start Buy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startSellingButton">
   <property name="geometry">
    <rect>
     <x>706</x>
     <y>0</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Sell</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startMatchingButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>0</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Matching</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startInverseBuyingButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Buy</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stockCodeLineEdit">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>0</y>
     <width>50</width>
     <height>18</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLineEdit" name="warrantCodeLineEdit">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>0</y>
     <width>50</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
